szperacz/logger.py

```diff
@@ -0,0 +1,19 @@
+
+
+class Logger:
+
+    def debug(self, msg_d):
+        return print(f'{DEBUG} {msg_d}')
+
+    def info(self, msg_i):
+        return print(f'{INFO} {msg_i}')
+
+    def error(self, msg_e):
+        return print(f'{ERROR} {msg_e}')
```

Comment:
Please use the same parameter for all methods: `msg`, remove those postfixes `*_d`, `_i`, `_e`